### ISSUE TITLE
Badges lack sufficient contrast

### ISSUE THREAD

**Issue by greatislander** [platform-coop-toolkit/pinecone#80]
## Describe the bug

[Badges](https://pinecone.platform.coop/components/detail/badges.html) lack sufficient contrast, according to [axe](https://www.deque.com/axe/):

> Element has insufficient color contrast of 4.31 (foreground color: #707070, background color: #f0efef, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

**Comment by dayotte** [comment]
@greatislander let's use a darker grey from our colour palette (#585850) for the foreground text - this passes WCAG AA for normal text at 6.25:1
